test/e2e/tests/contract-interactions.spec.js

```diff
@@ -81,10 +81,11 @@ describe('Deploy contract and call contract methods', function () {
         await driver.clickElement('#depositButton');
         await driver.waitUntilXWindowHandles(3);
         windowHandles = await driver.getAllWindowHandles();
-        await driver.switchToWindowWithTitle(
-          'MetaMask Notification',
-          windowHandles,
+        popup = windowHandles.find(
+          (handle) => handle !== extension && handle !== dapp,
         );
+        await driver.switchToWindow(popup);
```

Comment:
[Q] say what?

Reply:
I honestly don't know... For some reason the driver was no longer finding the window titled Metamask Notification, even though I could see that is still in fact the title of the window.

Reply:
Its possible firefox changed how our (and other extension) notifications are titled: <img width="362" alt="Screen Shot 2021-09-07 at 2 09 59 PM" src="https://user-images.githubusercontent.com/34557516/132398198-ad8de832-833d-4e8a-b714-9ae576ebee5f.png">

I think this change is more robust to these kinds of changes?

Reply:
But why would it only show up on this PR...

Reply:
i wonder if related to that double notification window close?

Reply:
oh yeah possibly. Unfortunately I can't confirm that atm because this test is dependent on gas estimation api availability  (which is [currently intermittent](https://consensys.slack.com/archives/GTQAGKY5V/p1631043080226300))

Reply:
That does appear to have been the issue 🤦 🤛  Pushed a commit to revert this change!